DIPs/accepted/DIP1018.md

```diff
@@ -674,4 +707,4 @@ One commenter asked about a default copy destructor. The DIP author replied that
 The language maintainers requested the Final Review round be dispensed with for this DIP.
 
 ### Formal Review
-The language maintainers considered the feature proposed here a top priority and worked with the DIP author to refine the implementation. When they were satisfied with the implementation, the DIP was given formal approval. The only change requested in the proposal was a comment that `A b = fun()` moves the result of `fun` into `b` be corrected, as `b` is constructed in place and no move is made.
\ No newline at end of file
+The language maintainers considered the feature proposed here a top priority and worked with the DIP author to refine the implementation. When they were satisfied with the implementation, the DIP was given formal approval. The only change requested in the proposal was a comment that `A b = fun()` moves the result of `fun` into `b` be corrected, as `b` is constructed in place and no move is made.
```

Comment:
What is the difference here?

Reply:
Original either has or lacked a newline at EOF or used a different line ending. Chuch the file into a hex editor, you'll see pretty quick.